utils/scripts/ocm/ocm.py

```diff
@@ -852,7 +852,7 @@ def update_notification_email_address(self, addon_name, email_address):
         if ret is None:
             log.info("Failed to update email address to {} addon on cluster "
                   "{}".format(addon_name, self.cluster_name))
-            sys.exit(1)
+            return False
```

Comment:
sys.exit I think is necessary when these API calls are done by Jenkins. So, I would suggest to give the option to use sys.exit or return, based on who is calling the function.
There are examples of what I'm suggesting in the same ocm.py

Reply:
We are not using jenkins API to directly executeg this method so why i didn't implement it. If you would like i can do that as well

Reply:
i have added it

Reply:
@tarukumar yeah I know, since it's an easy change I thought it is preferable to do that to maintain consistency with the resto of ocm.py. If in the future we are going to use the function in jenkins, we won't need any change